<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemAdminPage</class>
 <widget class="QWidget" name="SystemAdminPage">
  <property name="windowTitle">
   <string>ç³»ç»Ÿç®¡ç†å‘˜é¡µé¢</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_root">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>ç³»ç»Ÿç®¡ç†å‘˜åå°ç®¡ç†</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 24px; font-weight: bold; margin: 10px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_parkingLots"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <widget class="QPushButton" name="btn_refresh">
       <property name="text">
        <string>ğŸ”„ åˆ·æ–°</string>
       </property>
       <property name="minimumWidth">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>â• æ·»åŠ åœè½¦åœº</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>ğŸ—‘ åˆ é™¤åœè½¦åœº</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_queryViolation">
       <property name="text">
        <string>ğŸ“Š æŸ¥è¯¢è¿è§„æ•°æ®</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_buttons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_logout">
       <property name="text">
        <string>ğŸšª é€€å‡ºç™»å½•</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#e57373;color:white;border-radius:6px;padding:6px 12px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
